src/configs/settings.ts

```diff
@@ -205,6 +205,12 @@ export const PlanActivityTitles = [
   'BCC',
   'IRS',
 ] as const;
+
+export const goalsWithSymbols = [
```

Comment:
You need to add the values from the `planActivities` object e.g. `Larval_Dipping` should also be added to this list

Reply:
@FrankApiyo did you address this?

Reply:
Not yet. I'm on it.

Reply:
Is there a reason why you have not added `Mosquito_Collection`?